### PR TITLE
Add v1.3.0 perf and scale benchmarks data

### PR DESCRIPTION
**Pull request by Sreeja1725**
### What this PR does
Before this PR:
There is no doc for v1.3 
After this PR:
This PR adds  perf and benchmarks data doc for v1.3 release.

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
Add perf-scale benchmarks for release v1.3
```

### CODE REVIEW

File: docs/perf-scale-benchmarks.md [hunk 1]
```diff
@@ -37,10 +38,11 @@ taking specific assumptions into account. This section aims to provide clarity o
...
-1. The green dotted line in the graph is September 6, 2023, denoting change in k8s provider from v1.25 to v1.27
-1. The red dotted line in the graph is December 26, 2023, denoting release of v1.1.0
+1. The gray dotted line in the graph is July 6, 2023, denoting release of v1.0.0.
+1. The blue dotted line in the graph is September 6, 2023, denoting change in k8s provider from v1.25 to v1.27.
+1. The red dotted line in the graph is November 6, 2023, denoting release of v1.1.0.
+1. The green dotted line in the graph is March 5, 2024, denoting release of v1.2.0.
+1. The violet dotted line in the graph is March 25, 2024, denoting change in k8s provider from v1.27 to v1.29.
 
 
 #### Performance benchmarks for v1.2.0 release
```
Comment (alaypatel07) [inline docs/perf-scale-benchmarks.md:48]: s/v.1.2.0/v.1.3.0/g

**Review by alaypatel07 — CHANGES_REQUESTED** [review]
couple of nits, overall looks good

cc @xpivarc @rthallisey for reviews

File: docs/perf-scale-benchmarks.md [hunk 2]
```diff
@@ -37,10 +38,11 @@ taking specific assumptions into account. This section aims to provide clarity o
...
 1. The blue dots on the graphs are individual measurements, and orange line is weekly average
-1. The gray dotted line in the graph is March 1, 2023, denoting release of v0.60
-1. The blue dotted line in the graph is July 6, 2023, denoting release of v1.0.0
-1. The green dotted line in the graph is September 6, 2023, denoting change in k8s provider from v1.25 to v1.27
-1. The red dotted line in the graph is December 26, 2023, denoting release of v1.1.0
+1. The gray dotted line in the graph is July 6, 2023, denoting release of v1.0.0.
+1. The blue dotted line in the graph is September 6, 2023, denoting change in k8s provider from v1.25 to v1.27.
+1. The red dotted line in the graph is November 6, 2023, denoting release of v1.1.0.
+1. The green dotted line in the graph is March 5, 2024, denoting release of v1.2.0.
+1. The violet dotted line in the graph is March 25, 2024, denoting change in k8s provider from v1.27 to v1.29.
```
[the remark this answers is not in the document]
Comment (Sreeja1725) [reply docs/perf-scale-benchmarks.md:45]: Added

File: docs/perf-scale-benchmarks.md [hunk 3]
```diff
@@ -57,7 +59,9 @@ taking specific assumptions into account. This section aims to provide clarity o
 
 ![vmiCreationToRunningSecondsP95 for VM](perf-scale-graphs/vm/vm-p95-Creation-to-Running.png "vmiCreationToRunningSecondsP95 for VM")
 
-#### Scalability benchmarks for v1.2.0 release
+#### Scalability benchmarks for v1.3.0 release
+
+NOTE: The scalability metrics were hampered by a bug, that was fixed before the release [https://github.com//pull/12096]. Hence the scalability metrics data needs to be re-evaluated by users with the bug fix to get a more accurate benchmarks
```
Comment (alaypatel07) [inline docs/perf-scale-benchmarks.md:64]: broken link

**Comment by alaypatel07** [comment]
/lgtm

Thanks @Sreeja1725

**Comment by lyarwood** [comment]
PR looks great but a related question quickly while folks are looking at this for 1.3, I see the following still listed:

https://github.com/kubevirt/kubevirt/blob/2eb515574f9f963e9fe883f87f4bb5a31fa45cb9/docs/perf-scale-benchmarks.md#L22

Are there any plans to provide this? Is there anything I can do to help with this?

**Comment by alaypatel07** [comment]
@lyarwood yes there are a bunch of steps needed for getting the instance types metrics, your help will be very much appreciated. I think the best way would be to file an issue with sig/scale label so we can track it for 1.4 and discuss in sig-scale calls.

**Comment by alaypatel07** [comment]
@xpivarc / @lyarwood someone needs to give this PR an approve tag. Please take a look, thanks

**Comment by xpivarc** [comment]
@alaypatel07 @Sreeja1725 Do you want to re-run with https://github.com/kubevirt/kubevirt/pull/12249 ?

**Comment by alaypatel07** [comment]
@xpivarc the release benchmarks will not change since the fix will show data for 1.4 code due to branching. We don't have the ability to run on release branches yet.

**Comment by xpivarc** [comment]
> @xpivarc the release benchmarks will not change since the fix will show data for 1.4 code due to branching. We don't have the ability to run on release branches yet.

Thanks
/approve

**Pull request merged**